src/Akeneo/Pim/Enrichment/Product/back/API/Command/UserIntent/SetTableValue.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Pim\Enrichment\Product\API\Command\UserIntent;
+
+use Webmozart\Assert\Assert;
+
+/**
+ * @copyright 2022 Akeneo SAS (https://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+final class SetTableValue implements ValueUserIntent
+{
+    public function __construct(
```

Comment:
```suggestion
    /**
     * @param array<int, array<string, mixed>> $tableValue
     */
    public function __construct(
```

To explain what is expected. Don't forget the user intents are created for people that possibly does not know our bounded context.
You can also add an example if you want.